io.neovim.nvim.tool.tmux.yml

```diff
@@ -0,0 +1,56 @@
+app-id: io.neovim.nvim.tool.tmux
+branch: stable
+runtime: io.neovim.nvim
+runtime-version: stable
+sdk: org.freedesktop.Sdk//22.08
+build-extension: true
+separate-locales: false
+appstream-compose: false
+
+modules:
+  - name: libevent
+    sources:
+      - type: archive
+        url: https://github.com/libevent/libevent/releases/download/release-2.1.12-stable/libevent-2.1.12-stable.tar.gz
+        sha256: 92e6de1be9ec176428fd2367677e61ceffc2ee1cb119035037a27d346b0403bb
+    buildsystem: simple
+    build-commands:
+      - ./configure --prefix=${FLATPAK_DEST} --enable-shared
+      - make
+      - make install
+  - name: libutempter
+    sources:
+      - type: archive
+        url: https://github.com/altlinux/libutempter/archive/refs/tags/1.2.1-alt1.tar.gz 
+        sha256: 258d8bc4a42d2cbe4df6415912ab842574899adddbfc89d14ab56b57bb81158b 
+    buildsystem: simple
+    build-commands:
+      - make -C libutempter DESTDIR=${FLATPAK_DEST} libdir=/lib libexecdir=/lib includedir=/include
+      - make -C libutempter DESTDIR=${FLATPAK_DEST} libdir=/lib libexecdir=/lib includedir=/include install
+  - name: tmux
+    sources:
+      - type: archive
+        url: https://github.com/tmux/tmux/releases/download/3.3a/tmux-3.3a.tar.gz
+        sha256: e4fd347843bd0772c4f48d6dde625b0b109b7a380ff15db21e97c11a4dcdf93f
+    buildsystem: simple
+    build-commands:
+      - ./configure LIBRARY_PATH=${FLATPAK_DEST}/lib:$LIBRARY_PATH PKG_CONFIG_PATH=${FLATPAK_DEST}/lib/pkgconfig LDFLAGS=-L${FLATPAK_DEST}/lib CFLAGS=-I${FLATPAK_DEST}/include --prefix=${FLATPAK_DEST} --enable-utempter
+      - make
+      - make install
+  - name: tmux-wrapper
+    buildsystem: simple
+    sources:
+      - type: file
+        path: tmux-wrapper.sh
+    build-commands:
+      - install -Dm755 tmux-wrapper.sh /app/tools/tmux/bin/tmux-wrapper.sh
+      - chmod +x /app/tools/tmux/bin/tmux-wrapper.sh
```

Comment:
```suggestion
```
install does it for you

Reply:
Fixed: https://github.com/flathub/flathub/pull/4119/commits/107809a6d04da3056e1d8fd6f75e137e006249c0